integration-tests/features/environment.py

```diff
@@ -535,6 +546,7 @@ def before_all(context):
     jobs_api_url = _read_url_from_env_var('F8A_JOB_API_URL')
     gremlin_url = _read_url_from_env_var('F8A_GREMLIN_URL')
     threescale_url = _read_url_from_env_var('F8A_3SCALE_URL')
+    threescale_preview_url = _read_url_from_env_var('F8A_THREE_SCALE_PREVIEW_URL')
```

Comment:
dtto - the threescale_url is already set, not need to use preview

Reply:
`F8A_3SCALE_URL` and `F8A_THREE_SCALE_PREVIEW_URL` are two different services that's why I declared two variables, F8A_THREE_SCALE_PREVIEW service created specifically for the testing purpose.